src/chat.rs

```diff
@@ -3888,9 +3844,11 @@ mod tests {
             .unwrap();
         let msg = bob.get_last_msg().await;
 
+        println!("load from dbb");
```

Comment:
Load from "db"?

By the way, are these `printf` expected to be here or committed acidentally?